### ISSUE TITLE
[3.6.1] Request downloading an asset bundle,it cause stop main thread a while

### ISSUE THREAD

**Issue by dev-keishen** [cocos/cocos-engine#12781]
### Cocos Creator version

3.6.1

### System information

Android IOS

### Issue description

Request downloading an asset bundle,it cause stop main thread a while,it happen on Android IOS
This bug don't happen on previous version of 3.x

Please take a look into video : 
1. Press button and then request downloading an asset bundles
2. look into label : you will see count label will stop a while
https://user-images.githubusercontent.com/78413486/190346103-eeb9a6a5-fe7c-4004-99c7-9687a9d32753.mp4

### Relevant error log output

_No response_…

### Steps to reproduce

Please take a look into scene named scene and Test.ts component script

**Comment by dev-keishen** [comment]
@minggo 
I tried delete all sounds in asset bundles,it seem everything is normal again
I think it cause by loading huge sounds,it is same with this issue 
https://github.com/cocos/cocos-engine/issues/12776

This is a log : 

> Decoding (/data/user/0/com.game.kus/files/gamecaches/sandgame/16633182162015.wav) to pcm data wasted 886.760010ms

It take many time to decode an sound

I found this PR : https://github.com/cocos/cocos-engine/pull/12587/files#diff-970259cb145d4d5e37900e74d16eaeffbc3ce4785e9e6cb4c7b214bf0df4ab77

But it seem that it can not merged to 3.6.0 manually,is it right?

**Comment by minggo** [comment]
Yep, it is a big PR, not easy to merge manually. V3.6.1 will be release in days. You can have a try then.

**Comment by dev-keishen** [comment]
Will be fixed in 3.6.1

**Comment by dev-keishen** [comment]
In v3.6.1 this issue still happen on android and ios

Please take a look into video and notice counting label,when i press button to request downloading a bundle it stop main thread a while

https://user-images.githubusercontent.com/78413486/194821742-b57e0236-7878-4534-9a60-6039839b90d4.MP4

When i deleted all sounds in that bundle it is normal,i think this issues come from downloading and decoding sounds

I used cocos version 3.3.0 before,this issue don't happen

I would like to send a small project to reproduce : 

@minggo : Could you help me take a look into this issues?

**Comment by pandamicro** [comment]
New fix about Audio is here
https://github.com/cocos/cocos-engine/pull/12589

But be notice that asset bundle loading could include some very heavy main thread operation, like deserialization, instantiation of nodes, component wake up process etc. So the loading process is possible to block the main thread
